.travis.yml

```diff
@@ -142,6 +142,7 @@ before_script:
   # ycm
   - git clone https://github.com/robotology/ycm.git
   - cd ycm
+  - git checkout v0.10.4
```

Comment:
If you could comment here as to why we use this specific version, it'd be great.

Reply:
Done in https://github.com/robotology/icub-model-generator/pull/127/commits/722fae9d03414f0589bd43570c5ec5ae4065310e .